src/S3/MultipartCopy.php

```diff
@@ -108,7 +110,7 @@ protected function getUploadCommands(callable $resultHandler)
                 $command = $this->client->getCommand(
                     $this->info['command']['upload'],
                     $this->createPart($partNumber, $parts)
-                        + $this->getState()->getId()
+                    + $this->getState()->getId()
```

Comment:
Style nitpick: Can this parameter fit on one line (including the + component)?

Reply:
yes, it winds up being 67 characters